frontend/src/views/PublicCanteenStatisticsPage/index.vue

```diff
@@ -0,0 +1,379 @@
+<template>
+  <div class="text-left grey--text text--darken-4">
+    <h1 class="text-h4 font-weight-black black--text mt-3 mb-6">Découvrir les démarches chez vous</h1>
+    <!-- Add some introductory text? -->
+    <v-form class="my-4 pb-8">
+      <v-row>
+        <v-col cols="12" sm="6" md="4">
+          <label for="select-region" class="text-body-2">
+            Région
+          </label>
+          <v-autocomplete
+            v-model="chosenRegion"
+            :items="regions"
+            clearable
+            hide-details
+            id="select-region"
+            placeholder="Toutes les régions"
+            class="mt-1"
+            outlined
+            dense
+            auto-select-first
+            :filter="locationFilter"
+          ></v-autocomplete>
+        </v-col>
+        <v-col cols="12" sm="6" md="4">
+          <label for="select-department" class="text-body-2">
+            Département
+          </label>
+          <v-autocomplete
+            v-model="chosenDepartment"
+            :items="departments"
+            clearable
+            hide-details
+            id="select-department"
+            placeholder="Tous les départements"
+            class="mt-1"
+            outlined
+            dense
+            auto-select-first
+            :filter="locationFilter"
+          ></v-autocomplete>
+        </v-col>
+        <v-col cols="12" sm="6" md="4" class="d-flex align-end">
+          <v-btn x-large color="primary" @click="submit">
+            <!-- More descriptive text? -->
+            Rechercher
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-form>
+    <div v-if="locationText" class="py-8">
+      <h2 class="text--darken-5 text-h4 mb-8">Les statistiques pour {{ locationText }}</h2>
+      <v-row :class="{ 'flex-column': $vuetify.breakpoint.smAndDown }">
+        <v-col cols="12" md="6" class="pr-0">
+          <div id="published-canteen-text" class="mb-5">
+            <p>
```

Comment:
Je pense qu'on peut diminuer l'espace entre ces deux lignes (les deux parlent des cantines dans le territoire)
```suggestion
            <p class="mb-0">
```